extensions/kill-process/src/index.tsx

```diff
@@ -82,8 +93,14 @@ export default function ProcessList() {
   const killProcess = (process: Process) => {
     exec(`kill -9 ${process.id}`);
     setFetchResult(state.filter((p) => p.id !== process.id));
-    clearSearchBar({ forceScrollToTop: true });
-    showHUD(`✅ Killed ${process.processName === "-" ? `process ${process.id}` : process.processName}`);
+    if (!multipleKills) {
+      closeMainWindow();
+      popToRoot({ clearSearchBar: true })
```

Comment:
I think it is not necessary to add popToRoot, and this should respect to the the config in the "Advance" settings.

Reply:
I also agree with this, no need to pop.